menoh/mkldnn_with_generic_fallback/backend/mkldnn/operator/gemm.hpp

```diff
@@ -98,6 +94,7 @@ namespace menoh_impl {
                   gemm_output_md);
                 auto gemm_pd = mkldnn::inner_product_forward::primitive_desc(
                   gemm_desc, engine);
+                auto op_input_dims = extract_dims(gemm_pd.src_primitive_desc());
```

Comment:
`op_input_dims` is unused. Is it intentional?

Reply:
No. It was for debugging. I'll remove it.